README.md

```diff
@@ -64,7 +64,7 @@ This package requires the following to connect successfully:
   Client][client-role] role or equivalent. [Credentials](#credentials)
   for the IAM principal are used to authorize connections to an AlloyDB instance.
 
-* The [AlloyDB Admin API][admin-api] to be enabled within your Google Cloud
+* The [AlloyDB API][admin-api] to be enabled within your Google Cloud
```

Comment:
should we change the shortcut URL too so it isnt `admin-api`?

Reply:
updated